controller/upload_file.rst

```diff
@@ -362,6 +363,10 @@ automatically upload the file when persisting the entity::
             if ($file instanceof UploadedFile) {
                 $fileName = $this->uploader->upload($file);
                 $entity->setBrochure($fileName);
+            } elseif ($file instanceof File) {
+                // prevents the full file path being saved on updates
+                // as the path is set on the postLoad listener
+                $entity->setBrocure($file->getFilename());
```

Comment:
Typo: `setBrocure` -> `setBrochure`

Reply:
@javiereguiluz thank you for the feedback :)